web/modules/custom/dpl_webform/dpl_webform.install

```diff
@@ -107,3 +108,89 @@ function dpl_webform_update_10004(): string {
 
   return implode("\n", $feedback);
 }
+
+/**
+ * Set default language to english.
+ *
+ * We have experienced issues with the default site language being set to
+ * danish, which causes issues when doing configuration translations. To
+ * prevent this in the future, we now set the default language to english.
+ */
+function dpl_webform_update_10005(): string {
+  $feedback = [];
+
+  \Drupal::configFactory()->getEditable('system.site')->set('langcode', 'en')->save();
+  \Drupal::configFactory()->getEditable('system.site')->set('default_langcode', 'en')->save();
+
+  return implode("\n", $feedback);
```

Comment:
Why the empty feedback here? Maybe just retun empty string if it is expected that the update hooks returns a string?

Reply:
You are right. I have now added a string explaining what happened when running the hook.